vignettes/REDCapTidieR.Rmd

```diff
@@ -186,6 +187,60 @@ In summary, here are the rules by which `REDCapTidieR` constructs tibbles:
 
 -   The final field is always `form_status_complete`, which is an column to indicate whether the instrument was marked complete.
 
+## Adding variable labels with the `labelled` package
+
+`REDCapTidieR` integrates with the [{labelled}](http://larmarange.github.io/labelled/) package to allow you to add variable labels to your instrument tibbles. Let's load `labelled` to explore.
+
+```{r}
+library(labelled)
+```
+
+Use `make_labelled()` to apply labels to your supertibble. Let's also bind those tables into our global environment and use `labelled::look_for()` to see our labels. If you're using the RStudio IDE the labels also appear below your variable names in the viewer.
+
+```{r}
+make_labelled(superheroes_tidy) %>%
+  bind_tables()
+
+look_for(super_hero_powers)
+```
+
+Where did these labels come from? `make_labelled()` inferred them from the metadata stored in `superheroes_tidy$redcap_metadata`.
+
+```{r}
+superheroes_tidy %>%
+  filter(redcap_form_name == "super_hero_powers") %>%
+  unnest(redcap_metadata) %>%
+  select(field_name, field_label) %>%
+  glimpse()
+```
+
+The label for `power` doesn't look quite right. Let's remove that trailing "`:`". `make_labelled()` has a `format_labels` argument that we can use to preprocess labels in the metadata before applying them to the data.
+
+```{r}
+make_labelled(superheroes_tidy, format_labels = ~gsub(":", "", .)) %>%
+  bind_tables()
+
+look_for(super_hero_powers, "power")
+```
+
+This operation is common enough that `REDCapTidier` comes with a built-in helper function.
+
+```{r}
+fmt_strip_trailing_colon("Super Power:")
+```
+
+See the other available helpers with `` ?`format-helpers` ``.
+
+`format_labels` will accept any function that takes a vector of raw labels and returns a vector of formatted labels of the same length. You can also pass it a list functions to be executed on the raw labels in order.
```

Comment:
~"You can also pass it a list functions to be executed on the raw labels in order."~
There's an "of" missing in there, but I also think the order aspect is important to highlight. Suggestion:

"`format_labels` will also accept a list of functions, but it's important to note that the functions will be executed **in the order they are supplied**."